### ISSUE TITLE
Add Lightning to docs

### ISSUE THREAD

**Issue by s-ben** [decred/dcrdocs#946]
The technical documentation for our Lightning Network implementation lives in the dcrlnd [repo](https://github.com/decred/dcrlnd), as it should. However, now that LND is officially launched, it seems awkward to not have Lightning mentioned in the official docs. I propose adding at least a general overview page, which points to the technical docs and any other resources that might be useful.

**Comment by RichardRed0x** [comment]
I would like to see LN well documented, basically have as much info as we can about how it works, any differences to BTC/LTC, how to set up nodes and channels, etc.

**Comment by s-ben** [comment]
Thoughts on where a page would go? It doesn't seem to fit under any of the top menus. Just make a top-level `Lightning (LND)` menu?  Or `Layer 2` menu with Lightning subpage?

**Comment by jholdstock** [comment]
For now at least I would suggest adding it under the existing `Wallets` menu. Right now, LN is its own wallet, separate to dcrwallet. It might look the same because it is exposed via Decrediton, but they are totally independent tools.

**Comment by jholdstock** [comment]
Lets be careful with terminology as well. We have implemented Lightning Network (LN) with a binary named dcrlnd. lnd is the upstream project which we forked to create dcrlnd.

**Comment by s-ben** [comment]
Under `Wallets` makes sense then. I could create a decent high-level description of our LN implementation, but not familiar with the wallet aspects. Also wondering what level of technical detail is appropriate here wrt LN being in its own wallet.

**Comment by s-ben** [comment]
Taking a stab at a first draft.

**Comment by fguisso** [comment]
Hello guys, do you need some help?
Now with dcrwallet` implemented to `dcrlnd`, I think that `Lightning` needs an item on top-vel menu and a basic explain. How to run the CLI and how the network works is more for devdocs, what you think?

**Comment by s-ben** [comment]
Hey @fguisso, was just wondering about this actually. Dcrdocs is more user-facing. If LN is exposed via any interface (Decrediton or CLI) that is ready for actual usage, I'd say we should document that on the LN page ASAP. 

Advanced functionality only a dev would use, I'd say put it in devdocs.

**Comment by jholdstock** [comment]
I think we have two things to document here:

- General LN info and concepts - what its good for, how it works. This could be done at any time
- How to use DCR LN - CLI and/or Decrediton. Should only be done when available and ready for end-users

**Comment by s-ben** [comment]
Agreed. We could probably add general LN info and concepts now @fguisso if you want to take a stab.
